app/services/katello/pulp3/api/core.rb

```diff
@@ -55,6 +55,10 @@ def distributions_api
           repository_type.distributions_api_class.new(api_client)
         end
 
+        def untyped_repositories_api
```

Comment:
what does untyped mean?

Reply:
It's not specific to repository type like the `repositories_api`.  Perhaps I could call it `core_repositories_api` instead.

Reply:
that's what I was figured, I just wasn't sure. The name might be fine as is.